java/src/org/certificatetransparency/ctlog/SignedTreeHead.java

```diff
@@ -0,0 +1,16 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class SignedTreeHead {
+  public Ct.Version version;
+  public long timestamp;
+  public long treeSize;
+  public byte[] sha256RootHash;
+  public Ct.DigitallySigned signature;
+
+  public SignedTreeHead(Ct.Version version_) {
```

Comment:
what's the value in having a c'tor where you specify only one of SignedTreeHead's fields?  did you want to call out version explicitly for some reason?